dev/ms/helmfile.yaml

```diff
@@ -10,40 +10,34 @@ repositories:
 releases:
   - name: ms-jade-create-secret-manager-secret    # release name
     namespace: ms   # target namespace
-    createNamespace: true
```

Comment:
Should we make this change in all of personal dev helmfile.yaml's?

Reply:
yeah so the argo plugin was not a fan of that field